addons/osfstorage/models.py

```diff
@@ -342,6 +352,14 @@ def is_checked_out(self):
                 pass
         return False
 
+    @property
+    def is_preprint_primary(self):
+        if self.node.is_preprint:
+            for child in self.children:
```

Comment:
Deferring this change to https://openscience.atlassian.net/browse/OSF-7954 to not hold up this hotfix while waiting for travis.